gcmt/utils/interface.py

```diff
@@ -250,28 +255,31 @@ def set_data(self, time, tag=None, regrid_lowres=False):
         """
         self._set_data_common(time, tag=tag, regrid_lowres=False)
 
-        if self.ds.p_unit == 'bar':
-            press = self.ds.Z.values
-        elif self.ds.p_unit == 'Pa':
-            press = self.ds.Z.values/1e5
+        if self.dsi.p_unit == 'bar':
+            press = self.dsi.Z.values
+        elif self.dsi.p_unit == 'Pa':
+            press = self.dsi.Z.values / 1e5
         else:
-            raise NotImplementedError('only pressure units in Pa and bar are implemented at the moment')
+            raise NotImplementedError('only pressure units in Pa and bar are ' +
+                                      'implemented at the moment')
 
         # be careful here: petitRADTRANS operates top->bot in bar
-        self.pRT.setup_opa_structure(np.sort(press))
+        self.prt.setup_opa_structure(np.sort(press))
 
-    def calc_phase_spectrum(self, mmw, Rstar, Tstar, semimajoraxis, gravity=None, filename=None, normalize=True,
+    def calc_phase_spectrum(self, mmw, r_star, t_star, semimajoraxis, gravity=None,
+                            filename=None, normalize=True,
                             **prt_args):
         """
         Calculate the spectrum for a phasecurve.
 
         Parameters
         ----------
         mmw: float or 1D-array
-            Mean molecular weight (in atomic units). Will be globally uniform if float or horizonatally uniform if 1D.
-        Rstar: float
+            Mean molecular weight (in atomic units). Will be globally uniform if
+            float or horizonatally uniform if 1D.
+        r_star: float
             stellar radius in !cm!.
-        Tstar: float
+        t_star: float
```

Comment:
the issue is that Rstar and Tstar are the names that petitRADTRANS uses. I would opt to add a pylint ignore flag rather than chaning the names and introducing inconsistancies with prt

Reply:
I see the problem and do agree here. Generally, we should stay on standard coding notations, even if other libraries do not. My compromise would be to change it here but internally work with the correct notation.